content/en/docs/concepts/containers/images.md

```diff
@@ -33,17 +33,35 @@ Container images are usually given a name such as `pause`, `example/mycontainer`
 Images can also include a registry hostname; for example: `fictional.registry.example/imagename`,
 and possibly a port number as well; for example: `fictional.registry.example:10443/imagename`.
 
-If you don't specify a registry hostname, Kubernetes assumes that you mean the Docker public registry.
+If you don't specify a registry hostname, Kubernetes assumes that you mean the [Docker public registry](https://hub.docker.com/).
```

Comment:
I'm not sure if we need to specify a link to Dockerhub here but I guess it improves usability of this page. Reader can go straight to dockerhub to find something from default registry

Reply:
It is mentioned [here](https://kubernetes.io/docs/tasks/configure-pod-container/pull-image-private-registry/), so It is good to have that.